pokete_data/maps.py

```diff
@@ -268,7 +268,7 @@
     },
     "playmap_28": {
         "height": 55,
-        "width": 200,
+        "width": 198,
```

Comment:
Why do width -2?

Reply:
Because I realized the map was a bit too big.